### PR TITLE
week1-homework-API Abdulrahman sadek

### PR DESCRIPTION
**Pull request by Abdulrahmansadek**

### CODE REVIEW

File: 3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js [hunk 1]
```diff
@@ -12,8 +12,16 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(numbers) {
```
Comment (jontra000) [inline 3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js:15]: This function fails with the error message `Error(...) is not a constructor` when called with a value outside the range.

**Review by jontra000 — CHANGES_REQUESTED** [review]
Hi @Abdulrahmansadek, two issues to address before I can approve this.

- There is a bug in ex2
- You still need to add the explanations for ex3 and ex4 as requested at the end of the homework questions

Other than that, the code looks good!

**Review by Abdulrahmansadek — COMMENTED** [review]
@jontra000  thanks a lot for reviewing my code 
I fixed it now should be fine I think 
or ex2 the message it was out of the parentheses () 😣.
and I add explanation or ex3 and ex4

**Review by jontra000 — APPROVED** [review]
Great! All looks good now.

**Pull request closed without merging**